src/main/app/controller/courts/AddressController.ts

```diff
@@ -173,7 +179,11 @@ export class AddressController {
     errorMsgs: string[] = [],
     primaryPostcodeInvalid = false,
     secondaryPostcodeInvalid = false,
-    thirdPostcodeInvalid = false) {
+    thirdPostcodeInvalid = false,
+    primaryEpimInvalid = false,
```

Comment:
According to https://tools.hmcts.net/jira/browse/FACT-1903, it mentions that "The epim id can be seen as a new field on the Fact Admin Portal. When a user enters a value and saves, there is a confirmation of success that appears at the top of the page."

Therefore, we don't need more than one field added

We need to remove 

    secondaryEpimInvalid = false,
    thirdEpimInvalid = false
    
    From here and below

Reply:
changed